src/cdkutils/__init__.py

```diff
@@ -0,0 +1 @@
+from src.cdkutils.cdk_utils import PipelineConfig
```

Comment:
imports shouldn't contain the `src` dir as it's not part of the package
```suggestion
from .cdk_utils import PipelineConfig
```